internal/pkg/cli/app_delete.go

```diff
@@ -198,7 +164,6 @@ func (o *deleteAppOpts) deleteStacks() error {
 
 		stackName := fmt.Sprintf("%s-%s-%s", o.projectName, env.Name, o.AppName)
 
-		// TODO: check if the stack exists first
```

Comment:
:+1:

Reply:
it's done

Reply:
Sorry about the reply before, I think for this one it is no longer applies because if you don't have existing stack set it won't error out but just skip.